api/src/main/java/org/infinispan/api/annotations/indexing/model/Values.java

```diff
@@ -1,12 +1,22 @@
 package org.infinispan.api.annotations.indexing.model;
 
+import org.infinispan.api.annotations.indexing.option.VectorSimilarity;
+
 public final class Values {
 
    /**
     * This special value is reserved to not index the null value, that is the default behaviour.
     */
    public static final String DO_NOT_INDEX_NULL = "__Infinispan_indexNullAs_doNotIndexNull";
 
+   public static final int DEFAULT_INCLUDE_DEPTH = 3;
+
+   public static final VectorSimilarity DEFAULT_VECTOR_SIMILARITY = VectorSimilarity.L2;
+
+   public static final int DEFAULT_BEAN_WIDTH = 512;
+
+   public static final int DEFAULT_MAX_CONNECTIONS = 16;
```

Comment:
I'd recommend against explicit defaults like this, because values defined in annotations will override anything set in a `ValueBinder`. If you can't distinguish the default from "not set", you can't decide not to override whatever is set in the `ValueBinder`, and thus people cannot define custom vector field types in `ValueBinders`.

You could just use invalid values as defaults (e.g. -1, `Integer.MIN_VALUE`) like in Hibernate Search, and simply document the defaults in Javadoc.

Reply:
The problem is this I would like a default that is independent from the underlying layres.

Reply:
Then I guess configuring the backend defaults would be a better idea. That's not currently possible, but I suppose we could add that feature.

In the meantime, if you go with this... you know the consequences.